src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,10 +1,43 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
+type Props = {
+  onAdd: (movie: Movie) => void
+};
+
+export const NewMovie: React.FC<Props> = ({ onAdd }: Props) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+  const [count, setCount] = useState(0);
+  const [title, setTitle] = useState('');
+  const [imgUrl, setImgUrl] = useState('');
+  const [imdbUrl, setImdbUrl] = useState('');
+  const [imdbId, setImdbId] = useState('');
+  const [description, setDescription] = useState('');
```

Comment:
```suggestion
  const [movie, setMovie] = useState({
    title: '',
    imtUrl: '',
    imdbUrl: '',
    imdbId: '',
    description: ''
  });
```
consider to make one state for all field and use event.target.name for change proper field